components/clinical/flag-summary/src/atoms/flag-title.tsx

```diff
@@ -2,16 +2,26 @@ import { HTMLAttributes, FC } from 'react'
 import styled from '@emotion/styled'
 import { TEXT_COLOURS } from '@ltht-react/styles'
 import { Flag } from '@ltht-react/types'
-import { codeableConceptDisplaySummary } from '@ltht-react/utils'
+import { codeableConceptDisplaySummary, getStringExtension } from '@ltht-react/utils'
 
 const StyledFlagTitle = styled.div`
   color: ${TEXT_COLOURS.PRIMARY};
   text-align: left;
 `
+const StyledFlagTitleHighlight = styled.div`
+  color: ${TEXT_COLOURS.DANGER};
+  text-align: left;
+  padding-left: 2px;
+`
 
-const FlagTitle: FC<Props> = ({ flag: { code }, ...rest }) => (
-  <StyledFlagTitle {...rest}>{codeableConceptDisplaySummary(code)}</StyledFlagTitle>
-)
+const FlagTitle: FC<Props> = ({ flag: { code, extension }, ...rest }) => {
+  const ext = getStringExtension(extension, 'https://leedsth.nhs.uk/alert/priority')
+  if (ext === 'High') {
```

Comment:
This works without problem but does have a slight code smell in that it will unmount & mount a different component IF the priority flag is ever changed after initial render.

You could get around / "simplify" this to use a single emotion styled component by passing props through as such:

```
const StyledFlagTitle = styled.div<IStyledFlagTitleProps>`
  color:  ${TEXT_COLOURS.PRIMARY};
  text-align: left;
  ${(props) => props.priority === 'High' && `
    padding-left:2px;
    color: ${TEXT_COLOURS.DANGER};
  `
`

interface IStyledFlagTitleProps {
  priority: string
```

Reply:
Love that. Didn't know I could pass params to a styled comp like that. In this instance, the priority is set in reference data so will never change after render. But I've tweaked to be more concise.